src/Swift.Runtime/src/Metadata/ValueWitnessTable.cs

```diff
@@ -36,12 +36,10 @@ public ref struct ValueWitnessTable
 		public IntPtr AssignWithTake;
 		public IntPtr GetEnumTagSinglePayload;
 		public IntPtr StoreEnumTagSinglePayload;
-		private IntPtr _Size;
-		private IntPtr _Stride;
+		public nuint Size;
+		public nuint Stride;
 		public ValueWitnessFlags Flags;
 		public uint ExtraInhabitantCount;
-        public int Size => _Size.ToInt32();
-        public int Stride => _Stride.ToInt32();
 		public int Alignment => (int)((Flags & ValueWitnessFlags.AlignmentMask) + 1);
 		public bool IsNonPOD => Flags.HasFlag (ValueWitnessFlags.IsNonPOD);
```

Comment:
```suggestion
		public bool IsNonPOD => (Flags & ValueWitnessFlags.IsNonPOD) != 0;
```
It is better to use simple bit test rather than `HasFlag`. `HasFlag` will get optimized most of the time to a bit test, but not all the time. For example, `HasFlag` won't get optimized in Tier0 that is bad for startup time.